gui-apps/foot/foot-1.10.3.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson xdg
+
+if [[ ${PV} != *9999* ]]; then
+	SRC_URI="https://codeberg.org/dnkl/foot/archive/${PV}.tar.gz  -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/${PN}"
+else
+	inherit git-r3
+	EGIT_REPO_URI="https://codeberg.org/dnkl/foot.git"
+fi
+
+DESCRIPTION="A fast, lightweight and minimalistic Wayland terminal emulator"
+HOMEPAGE="https://codeberg.org/dnkl/foot"
+LICENSE="MIT"
+SLOT="0"
+IUSE="ime +themes +grapheme-clustering"
```

Comment:
I feel like `themes` doesn't need to be a USE, no dependencies and sounds it's just installing a few extra small files that fall under:
https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0301

And does `ime` really have no dependencies? I tend to wonder about USE that only toggle flags without anything else. Unless there's a reason people want to disable something (less dependencies / undesirable behavior / significant size or build time changes) it often make sense to just enable it.